package.json

```diff
@@ -3,13 +3,17 @@
   "version": "0.0.0",
   "scripts": {
     "clean:game": "rm -r saves/*.json",
-    "compile:schemas": "ts-node scripts/denormalizeSchemas.ts && json2ts --cwd=src/utils/schemas/normalized/ src/utils/schemas/normalized/gameState.json > src/utils/types/gameState.d.ts && ts-node scripts/compileRoomTypesTuple.ts && prettier --loglevel silent --write src/utils/types/gameState.d.ts src/utils/types/roomTypesTuple.ts",
-    "dev:analyze": "npm run compile:schemas && npm run lint:ts && npm run typecheck",
-    "dev:test": "npm run compile:schemas && npm run test:src && npm run test:e2e",
+    "dev:analyze": "npm run schemas:update && npm run lint:ts && npm run typecheck",
+    "dev:test": "npm run schemas:update && npm run test:src && npm run test:e2e",
     "klicker-knight": "ts-node src/klickerKnight",
     "lint:md": "markdownlint **/*.md --ignore node_modules",
     "lint:ts": "eslint --color --ext .ts,.js .",
     "lint:yaml": "yamllint --ignore=node_modules/**/*.yaml **/*.yaml",
+    "schemas:compile-base-types": "json2ts --cwd=src/utils/schemas/normalized/ src/utils/schemas/normalized/gameState.json > src/utils/types/gameState.d.ts",
+    "schemas:compile-custom-types": "ts-node scripts/compileRoomTypesTuple.ts",
```

Comment:
I renamed the script "compileRoomTypesTuple" to "compileCustomTypes" in the next commit